exampleCourse/questions/demo/workspace/vscode-cpp/question.html

```diff
@@ -0,0 +1,17 @@
+<pl-question-panel> 
+    <p>Write a program that outputs the perimeter of a rectangle of a user-provided integer length and width.
+        Your output must match the example exactly, including the whitespace and the newline character at the end.
+    </p>
+    <pl-code language="c++">
```

Comment:
This isn't really C++; maybe `text` or `output` would be more appropriate? https://pygments.org/languages/

Reply:
I think that was copied from somewhere else. There isn't any `pl-code` element in the Python workspace demo, so I'll take it out completely.

Reply:
Looks like this was resolved in 3a55d7a.